src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -39,17 +42,20 @@ internal TagHelperExecutionContext(string tagName)
         /// <param name="executeChildContentAsync">A delegate used to execute the child content asynchronously.</param>
         /// <param name="startWritingScope">A delegate used to start a writing scope in a Razor page.</param>
         /// <param name="endWritingScope">A delegate used to end a writing scope in a Razor page.</param>
+        /// <param name="parentItems">Gets the collection of items used to communicate with child <see cref="ITagHelper"/>s</param>
         public TagHelperExecutionContext([NotNull] string tagName,
                                          [NotNull] string uniqueId,
                                          [NotNull] Func<Task> executeChildContentAsync,
                                          [NotNull] Action startWritingScope,
-                                         [NotNull] Func<TextWriter> endWritingScope)
+                                         [NotNull] Func<TextWriter> endWritingScope,
+                                         [NotNull] IDictionary<object, object> parentItems)
```

Comment:
nit: suggest placing this parameter above the various `Func` and `Action` parameters, as you've done in the `TagHelperContext` constructor (for complete consistency, between the `tagName` and `uniqueId` parameters)